use -fopenmp rather than -openmp, as required by recent versions of the Intel compilers
author: Kenneth Hoste (HPC-UGent)
--- DIRAC-18.0-Source/cmake/downloaded/autocmake_omp.cmake.orig	2019-07-08 18:35:35.338342044 +0200
+++ DIRAC-18.0-Source/cmake/downloaded/autocmake_omp.cmake	2019-07-08 18:36:03.818957830 +0200
@@ -52,7 +52,7 @@
                 set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -Qopenmp")
             elseif(CMAKE_Fortran_COMPILER_ID STREQUAL "Intel" AND
                    "${CMAKE_Fortran_COMPILER_VERSION}" VERSION_LESS "15.0.0.20140528")
-                set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -openmp")
+                set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fopenmp")
             else()
                 set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -qopenmp")
             endif()
--- DIRAC-18.0-Source/cmake/downloaded/autocmake_math_libs.cmake.orig	2019-07-08 18:35:29.248212301 +0200
+++ DIRAC-18.0-Source/cmake/downloaded/autocmake_math_libs.cmake	2019-07-08 18:36:21.679328011 +0200
@@ -369,7 +369,7 @@
         set(_omp_flag)
         if(HAVE_MKL_BLAS OR HAVE_MKL_LAPACK)
             if(MKL_COMPILER_BINDINGS MATCHES Intel)
-                set(_omp_flag -openmp)
+                set(_omp_flag -fopenmp)
             endif()
             if(MKL_COMPILER_BINDINGS MATCHES GNU)
                 set(_omp_flag -fopenmp)
